.docker/docker-compose.yml

```diff
@@ -6,3 +6,12 @@ services:
       context: ../
     ports:
       - '8080:8080'
+  console:
+    image: tpluscode/hypermedia-app:alpha
```

Comment:
Will need a tag released before can merge. /cc @tpluscode

Reply:
Heh, hadn't noticed https://hub.docker.com/layers/tpluscode/hypermedia-app/version-0.12.0/images/sha256-dc1a9ddf92da97461275fd819a0e5029ae8203027f4e3f1e9205ff5c8b9227b9 was pushed. 🎉

Reply:
Unfortunately it's still one with the environment variables not honoured (hypermedia-app/generic.hypermedia.app#44). Stay tuned for build 0.12.1

Reply:
> docker pull tpluscode/hypermedia-app:version-0.12.1